pkg/lib/listing.less

```diff
@@ -222,6 +222,7 @@ tr.listing-ct-item > td {
 
 /* Listing actions can be used directly as a cell */
 tr.listing-ct-item td.listing-ct-actions,
+caption div.listing-ct-actions,
```

Comment:
I see caption listed here. Are captions used? They're sometimes buggy across browsers, _especially_ with WebKit-based browsers.

Reply:
Actually it's not used - sorry for the confusion - I should remove this.